specificatie/immigratie.yaml

```diff
@@ -0,0 +1,59 @@
+openapi: 3.0.3
+info:
+  title: Immigratie definities
+  version: 2.0.0
+  contact: {}
+paths: {}
+components:
+  schemas:
+    IndicatieVestigingVanuitBuitenland:
+      type: boolean
+      description: |
+        Geeft aan dat de persoon zich vanuit het buitenland heeft ingeschreven.
+    VanuitVerblijfplaatsOnbekend:
+      type: boolean
+      description: |
+        Geeft aan dat de persoon is teruggekeerd uit een situatie van onbekende verblijfplaats
+    GbaImmigratie:
+      type: object
+      description: |
+        Gegevens over het het immigratie van een persoon.
+          * **landVanWaarIngeschreven** : het land waar de persoon woonde voor (her)vestiging in Nederland.
+      properties:
+        datumVestigingInNederland:
+          $ref: 'datum.yaml#/components/schemas/AbstractDatum'
+        landVanwaarIngeschreven:
+          $ref: 'common.yaml#/components/schemas/Waardetabel'
+        vanuitVerblijfplaatsOnbekend:
```

Comment:
vanuitVerblijfplaatsOnbekend kan niet in GbaImmigratie zitten. Dit is een afgeleid gegeven.

Reply:
vanuitVerblijfplaatsOnbekend verwijderd uit GbaImmigratie